web/src/auth/mfa/MfaAuthVerifyForm.js

```diff
@@ -71,20 +71,29 @@ export function MfaAuthVerifyForm({formValues, authParams, mfaProps, application
         <div style={{marginBottom: 24, textAlign: "center", fontSize: "24px"}}>
           {i18next.t("mfa:Multi-factor authentication")}
         </div>
-        <div style={{marginBottom: 24}}>
-          {i18next.t("mfa:You have enabled multi-factor authentication, please enter the authentication code")}
-        </div>
         {mfaType === SmsMfaType || mfaType === EmailMfaType ? (
-          <MfaVerifySmsForm
-            mfaProps={mfaProps}
-            method={mfaAuth}
-            onFinish={verify}
-            application={application}
-          />) : (
-          <MfaVerifyTotpForm
-            mfaProps={mfaProps}
-            onFinish={verify}
-          />
+          <div>
+            <div style={{marginBottom: 24}}>
+              {i18next.t(
+                "mfa:You have enabled Multi-factor Authentication. Please click Send Code and follow the prompts to finish your login process"
+              )}
+            </div>
+            <MfaVerifySmsForm
+              mfaProps={mfaProps}
+              method={mfaAuth}
+              onFinish={verify}
+              application={application}
+            />
+          </div>
+        ) : (
+          <div>
+            <div style={{marginBottom: 24}}>
+              {i18next.t(
+                "mfa:You have enabled multi-factor authentication, please enter the authentication code"
```

Comment:
Use: `"mfa:You have enabled Multi-Factor Authentication, please enter the TOTP code"`

Reply:
This sounds better to me as well. However, this is an existing message so I hope it is okay if I update it.

Reply:
As I am updating the existing message, there are a couple of more minor enhancements in MFA dialog that I see and I will make those changes as well.

1. Changing "Multi-falcon authentication" to "Multi-factor Authentication" in the heading. 
2. Adding a space between `Having a problem` and `Use recovery code`

Please let me know if these two sounds like a good improvement to you @hsluoyz @dacongda

Reply:
Both OK